### DISCUSSION TITLE
Further development for iGPU's (680M)

### DISCUSSION THREAD

**Discussion by hpm04** [ChefKissInc/NootRX#53] [Q&A] [answered]
Hey everyone, Im new to hackintoshing and using Github in general, but i was wondering how could i get access to the thought process behind making the kext work for rdna2 gpu's. I have a 680m i would like to use for a build with graphics accelaration but its yet not supported by neither nootedred/ nootrx. My question is, how can i start modifying the kext and incorporate the 680m to work with it? i know this is very challenging but i am very curious to how it works and would like resources and tips on where to begin. Thank you.

**Comment by VisualEhrmanntraut** [comment, marked as answer]
This is strictly for AMD dGPUs. I don't know if I'll do it, because the RDNA iGPUs are a lot of effort, and I don't know if you realise but I'm the only developer.
